d2planner/src/formatters.js

```diff
@@ -68,7 +70,7 @@ function createFillTaWithCalcAFormatter (pattern) {
 }
 
 
-function createCalcFormatter (template, frames=false, gameUnits=false, precision=null, multiplier=null) {
+function createCalcFormatter ({template, frames=false, gameUnits=false, precision=null, multiplier=null}={}) {
```

Comment:
Here's the descructuring hack, looks ok here, but requires an object to be passed where it's called. Thankfully it's only called in this file.